cmd/kubeconfig/create.go

```diff
@@ -91,7 +92,7 @@ func createUserKubeconfig(k0sVars *config.CfgVars, clusterAPIURL, username, grou
 	certManager := certificate.Manager{
 		K0sVars: k0sVars,
 	}
-	userCert, err := certManager.EnsureCertificate(userReq, users.RootUID)
+	userCert, err := certManager.EnsureCertificate(userReq, users.RootUID, 8760*time.Hour)
```

Comment:
Would be cool if we'd get a flag for this maybe? E.g. `--certificate-expires-after` or sth. along those lines?

Reply:
All comments addressed, PTAL